Servers/utils/question.utils.ts

```
@@ -1,17 +1,13 @@
 import { sequelize } from "../database/db";
 import { deleteFileById, getFileById } from "./fileUpload.utils";
 import { Request } from "express";
 import { QueryTypes, Transaction } from "sequelize";
+import { IQuestion } from "../domain.layer/interfaces/I.question";
 
 export const getAllQuestionsQuery = async (
   tenant: string
-): Promise<
-  (Question & { evidence_files: Object[] })[]
-> => {
+): Promise<(IQuestion & { evidence_files: Object[] })[]> => {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)